floor_generator/lib/processor/entity_processor.dart

```diff
@@ -266,6 +266,8 @@ class EntityProcessor extends Processor<Entity> {
       return '$parameterValue as String';
     } else if (parameterType.isDartCoreInt) {
       return '$parameterValue as int';
+    } else if (parameterType.getDisplayString() == 'Uint8List') {
+      return '$parameterValue';
```

Comment:
💡Please make sure to cast the parameter value to `Uint8List` here.

Reply:
I tried using `$parameterValue as Uint8List`, but because Uint8List is not a concrete type but an abstract class, it won't let me cast into it... is there a special way I have to take?

Reply:
```suggestion
      return '$parameterValue as Uint8List';
```

Works for me

Reply:
You're right! I must have missed something when I tried it...